projects/extension/ai/load_dataset.py

```diff
@@ -58,30 +66,57 @@ def create_table(
         # Replace any non-alphanumeric characters with underscore
         table_name = "".join(c if c.isalnum() else "_" for c in base_name.lower())
 
-    # Construct fully qualified table name
-    qualified_table = f'"{schema}"."{table_name}"'
+        # Check table name length - PostgreSQL has a 63 character limit for identifiers
+        if len(table_name) > 63:
```

Comment:
I think it's actually a 63 octet limit, which *should be the same unless someone starts using non-ascii characters, right?